### PR TITLE
feat: payment method list

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/features/metadata/constants.js [hunk 1]
```diff
@@ -35,5 +35,41 @@ export const DELETE_OFFER_SUCCEEDED = 'METADATA/DELETE_OFFER_SUCCEEDED';
 export const DELETE_OFFER_FAILED = 'METADATA/DELETE_OFFER_FAILED';
 
 // Mapping
-export const PAYMENT_METHODS = ['Cash (In person)', 'Bank Transfer', 'International wire'];
+export const POPULAR_PAYMENT_METHODS = [1, 2, 3];
```
Comment (jrainville) [inline src/js/features/metadata/constants.js]: Could you rename this `POPULAR_PAYMENT_METHODS_INDEXES` so it's clear that this list is actually just the indexes and not the actually methods

File: src/js/features/metadata/constants.js [hunk 2]
```diff
@@ -35,5 +35,41 @@ export const DELETE_OFFER_SUCCEEDED = 'METADATA/DELETE_OFFER_SUCCEEDED';
...
+    23: 'Skrill',
+    24: 'Swish',
+    25: 'TransferWise',
+    26: 'UPI',
+    27: 'Venmo',
+    28: 'WebMoney',
+    29: 'WeChat Pay',
+    30: 'Western Union',
+    31: 'Yandex.Money',
+    99999: 'Other'
```
Comment (jrainville) [inline src/js/features/metadata/constants.js]: lol

File: src/js/pages/OffersList/components/SorterFilter.jsx [hunk 3]
```diff
@@ -62,18 +62,28 @@ class FilterMenu extends Component {
...
+                            key={'paymentMethod-' + index} isCheckBox
+                            onClick={(_e) => props.setPaymentMethodFilter(index)}>
+                  {PAYMENT_METHODS[index]}
                 </CheckButton>
               ))}
             </ButtonGroup>
 
+            <span className="text-muted text-small mt-3">All payment methods (A-Z)</span>
+            <ButtonGroup vertical className="w-100">
+              {Object.keys(PAYMENT_METHODS).map(x => parseInt(x, 10)).filter(x => POPULAR_PAYMENT_METHODS.indexOf(x) === -1).map((index) => (
```
Comment (jrainville) [inline src/js/pages/OffersList/components/SorterFilter.jsx]: Why do you do two `map`s? The first one seems useless.

File: src/js/wizards/Sell/1_PaymentMethods/components/SellerPaymentMethod.jsx [hunk 4]
```diff
@@ -3,23 +3,35 @@ import {ButtonGroup} from 'reactstrap';
...
+              {PAYMENT_METHODS[idx]}
+            </CheckButton>
+          ))}
+        </ButtonGroup>
 
-        <ButtonGroup vertical className="w-100 mt-3">
-          {this.props.methods.map((asset, idx) => (
+        <span className="text-muted text-small mt-3">All payment methods (A-Z)</span>
+        <ButtonGroup vertical className="w-100">
+          {Object.keys(PAYMENT_METHODS).map(x => parseInt(x, 10)).filter(x => POPULAR_PAYMENT_METHODS.indexOf(x) === -1).map((idx) => (
```
Comment (jrainville) [inline src/js/wizards/Sell/1_PaymentMethods/components/SellerPaymentMethod.jsx]: Again, I think the first `map` is not really needed

Comment (jrainville) [reply src/js/wizards/Sell/1_PaymentMethods/components/SellerPaymentMethod.jsx]: Worst case you can do `.filter(x => POPULAR_PAYMENT_METHODS.indexOf(parseInt(x, 10)) === -1)`
instead, that way you don't loop again

File: src/js/pages/OffersList/components/SorterFilter.jsx [hunk 5]
```diff
@@ -64,9 +64,9 @@ class FilterMenu extends Component {
             <h5 className="mt-4">Payment method</h5>
             <span className="text-muted text-small">Popular</span>
             <ButtonGroup vertical className="w-100">
-              {POPULAR_PAYMENT_METHODS.map((index) => (
+              {POPULAR_PAYMENT_METHODS_INDEXES.map((index) => (
                 <CheckButton active={index === props.paymentMethodFilter}
-                            key={'paymentMethod-' + index} isCheckBox
```
Comment (jrainville) [inline src/js/pages/OffersList/components/SorterFilter.jsx]: Why remove isCheckBox?

Comment (richard-ramos) [reply src/js/pages/OffersList/components/SorterFilter.jsx]: they were originally radio buttons, and due to copy-paste I turned them into a checkbox. With this, I'm returning it back to their correct format.

Comment (jrainville) [reply src/js/pages/OffersList/components/SorterFilter.jsx]: Well, they should be check boxes though. Check the design and also the code, we can select multiple ones

Comment (richard-ramos) [reply src/js/pages/OffersList/components/SorterFilter.jsx]: Even in the selection filter for the offer list?

Comment (jrainville) [reply src/js/pages/OffersList/components/SorterFilter.jsx]: Oh... 🤦‍♂ 
My bad I hadn't realized this was the filter. The filter should be radio, but the seller page checkbox

**Pull request merged**